### PR TITLE
Add penpot to catalog

### PR DESCRIPTION
**Pull request by orhtej2**
Waiting for CI.....

### CODE REVIEW

File: apps.toml [hunk 1]
```diff
@@ -2465,6 +2465,11 @@ state = "working"
 subtags = [ "website" ]
 url = "https://github.com/YunoHost-Apps/pelican_ynh"
 
+[penpot]
+category = "productivity_and_management"
```
Comment (OniriCorpe) [inline apps.toml:2469]: i think `dev` would be more suitable

File: logos/penpot-logo-icon.png
Comment (ericgaspar) [inline logos/penpot-logo-icon.png:1]: This should be named `penpot.png`(there is already an existing logo file in `logos` folder )

File: apps.toml (hunk 1 above)
Comment (ericgaspar) [reply apps.toml:2469]: `productivity_and_management` seems fine to me.

File: logos/penpot-logo-icon.png
Comment (orhtej2) [reply logos/penpot-logo-icon.png:1]: 🤦

File: apps.toml (hunk 1 above)
Comment (OniriCorpe) [reply apps.toml:2469]: But it’s literally a dev tool, to design apps or websites x)

Comment (Salamandar) [reply apps.toml:2469]: Well i kinda disagree, i'd say it's closer to a drawing too than to a development tool ^^''

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**